protostar-rust/tests/data/dispatchers/tests/using_dispatchers.cairo

```diff
@@ -15,7 +15,7 @@ use dispatchers::hello_starknet::IHelloStarknetDispatcherTrait;
 #[test]
 fn call_and_invoke() {
     let class_hash = declare('HelloStarknet').unwrap();
-    let prepared = PreparedContract { contract_address: 1234, class_hash: class_hash, constructor_calldata: @ArrayTrait::new() };
+    let prepared = PreparedContract { contract_address: 1234, class_hash: class_hash, constructor_calldata: ArrayTrait::new() };
```

Comment:
Why are we no longer using snapshots here (`@`)? We specifically changed our code to use them sometime ago.